plugins/ua_accesscontrol_default.c

```diff
@@ -253,6 +253,7 @@ UA_AccessControl_default(UA_ServerConfig *config, UA_Boolean allowAnonymous,
     UA_LOG_WARNING(&config->logger, UA_LOGCATEGORY_SERVER,
                    "AccessControl: Unconfigured AccessControl. Users have all permissions.");
     UA_AccessControl *ac = &config->accessControl;
+    clear_default(ac);
```

Comment:
We don't know if the previous AccessControl config is from the same plugin.

Here you should check if `ac->clear` is already set. If yes, use that to clean up.
Otherwise, there is no need to clear.